frame/mosaic/src/lib.rs

```diff
@@ -183,6 +188,21 @@ pub mod pallet {
 	#[allow(clippy::disallowed_types)]
 	pub type Nonce<T: Config> = StorageValue<_, u128, ValueQuery>;
 
+	/// Remote AMM IDs that exist (NetworkId, AmmId).
+	/// Note that this is actually a set that does bookkeeping of valid AmmIds.
+	/// Therefore, the value type is (), because it is irrelevant for our use case.
+	#[pallet::storage]
+	#[pallet::getter(fn amm_ids)]
+	pub type RemoteAmmIds<T: Config> = StorageDoubleMap<
```

Comment:
How about RemoteAmmWhitelist? As this type is not used to keep track of remote amms, but of locally accepted IDs?

Reply:
I like it, will do